commands/ThresholdCommand.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc4904.standard.commands;
+
+
+import java.util.function.Supplier;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ThresholdCommand extends Command {
+	protected final Command command;
+	protected final Supplier<Double> axis;
+	protected final double threshold;
+	protected final boolean invert;
+
+	public ThresholdCommand(Command command, Supplier<Double> axis, double threshold, boolean invert) {
+		this.command = command;
+		this.axis = axis;
+		this.threshold = threshold;
+		this.invert = invert;
+	}
+
+	public ThresholdCommand(Command command, Supplier<Double> axis, double threshold) {
+		this(command, axis, threshold, false);
+	}
+
+	protected boolean pastThreshold(double value) {
+		return (!invert && value >= threshold) || (invert && value <= threshold);
```

Comment:
This is pedantic, but I think we can do this without doing the `value >= threshold` twice:
```
(value >= threshold) ^ invert
```
where the `^` is XOR (exclusive or, one or the other but not both).
We want to return true if the value is greater than the threshold or if we're inverted but not both.